doc/encoders.texi

```diff
@@ -3356,6 +3356,16 @@ Change this value to reset qsv codec's gop configuration.
 @item @var{int_ref_qp_delta}
 @item @var{int_ref_cycle_dist}
 Change these value to reset qsv codec's Intra Refresh configuration.
+
+@item @var{qmax}
+@item @var{qmin}
+@item @var{max_qp_i}
+@item @var{min_qp_i}
+@item @var{max_qp_p}
+@item @var{min_qp_p}
+@item @var{max_qp_b}
+@item @var{min_qp_b}
+Change these value to reset qsv codec's max/min qp configuration.
```

Comment:
applicability missing:
```
Supported in h264_qsv and hevc_qsv.
```

Reply:
Done, thanks

Reply:
I verified it only supported on h264, so only add it on h264.

Reply:
Also confirmed with VPL team.